commit 05f8a44b83a01ab60774a14b48c39067dc135171
Author: Endi Sukma Dewata <edewata@redhat.com>
Date:   Wed Feb 22 01:15:56 2012 -0600

    Removed dependency on OSUtil.
    
    The OSUtil is no longer used by the code. It has been removed from
    build scripts and tools.
    
    Ticket #90

--- a/pki/.classpath
+++ b/pki/.classpath
@@ -25,7 +25,6 @@
 	<classpathentry kind="lib" path="/usr/share/java/ldapjdk.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/idm-console-nmclf.jar"/>
 	<classpathentry kind="lib" path="/usr/lib/java/jss4.jar"/>
-	<classpathentry kind="lib" path="/usr/lib/java/osutil.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/velocity.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/servlet.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/xerces-j2.jar"/>
--- a/pki/base/CMakeLists.txt
+++ b/pki/base/CMakeLists.txt
@@ -1,7 +1,6 @@
 project(base)
 
 # The order is important!
-#   add_subdirectory(osutil)
 if (APPLICATION_FLAVOR_PKI_CORE)
     add_subdirectory(setup)
     add_subdirectory(symkey)
--- a/pki/base/ca/src/CMakeLists.txt
+++ b/pki/base/ca/src/CMakeLists.txt
@@ -17,13 +17,6 @@
         ${JAVA_LIB_INSTALL_DIR}
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -49,7 +42,7 @@
     ${PKI_CERTSRV_JAR} ${PKI_CMS_JAR} ${PKI_CMSCORE_JAR}
     ${PKI_CMSUTIL_JAR} ${PKI_NSUTIL_JAR}
     ${LDAPJDK_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
 
 
 # set version
@@ -58,7 +51,7 @@
 
 # build pki-ca.jar
 add_jar(pki-ca ${pki-ca_java_SRCS})
-add_dependencies(pki-ca osutil symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
+add_dependencies(pki-ca symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
 install_jar(pki-ca ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_CA_JAR ${pki-ca_JAR_FILE} CACHE INTERNAL "pki-ca jar file")
 
--- a/pki/base/common/src/CMakeLists.txt
+++ b/pki/base/common/src/CMakeLists.txt
@@ -16,14 +16,6 @@
         /usr/share/java
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-        /usr/share/java
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -1002,21 +994,21 @@
 set(CMAKE_JAVA_INCLUDE_PATH
     ${PKI_NSUTIL_JAR} ${PKI_CMSUTIL_JAR}
     ${LDAPJDK_JAR} ${SERVLET_JAR} ${VELOCITY_JAR} ${XALAN_JAR} ${XERCES_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
 
 set(CMAKE_JAVA_TARGET_VERSION ${APPLICATION_VERSION})
 
 # build pki-certsrv
 set(CMAKE_JAR_CLASSES_PREFIX com/netscape/certsrv)
 add_jar(pki-certsrv ${pki-certsrv_java_SRCS})
-add_dependencies(pki-certsrv osutil pki-nsutil pki-cmsutil)
+add_dependencies(pki-certsrv pki-nsutil pki-cmsutil)
 install_jar(pki-certsrv ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_CERTSRV_JAR ${pki-certsrv_JAR_FILE} CACHE INTERNAL "pki-certsrv jar file")
 
 # build pki-cms
 set(CMAKE_JAR_CLASSES_PREFIX com/netscape/cms)
 add_jar(pki-cms ${pki-cms_java_SRCS})
-add_dependencies(pki-cms osutil pki-nsutil pki-cmsutil pki-certsrv)
+add_dependencies(pki-cms pki-nsutil pki-cmsutil pki-certsrv)
 install_jar(pki-cms ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_CMS_JAR ${pki-cms_JAR_FILE} CACHE INTERNAL "pki-cms jar file")
 
@@ -1034,12 +1026,12 @@
 # build pki-cmscore
 set(CMAKE_JAR_CLASSES_PREFIX com/netscape/cmscore)
 add_jar(pki-cmscore ${pki-cmscore_java_SRCS})
-add_dependencies(pki-cmscore osutil pki-nsutil pki-cmsutil pki-certsrv pki-cms)
+add_dependencies(pki-cmscore pki-nsutil pki-cmsutil pki-certsrv pki-cms)
 install_jar(pki-cmscore ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_CMSCORE_JAR ${pki-cmscore_JAR_FILE} CACHE INTERNAL "pki-cmscore jar file")
 
 # build pki-cmsbundle
 add_jar(pki-cmsbundle ${pki-cmsbundle_RCS})
-add_dependencies(pki-cmsbundle osutil pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
+add_dependencies(pki-cmsbundle pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
 install_jar(pki-cmsbundle ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_CMSBUNDLE_JAR ${pki-cmsbundle_JAR_FILE} CACHE INTERNAL "pki-cmsbundle jar file")
--- a/pki/base/console/src/CMakeLists.txt
+++ b/pki/base/console/src/CMakeLists.txt
@@ -69,14 +69,6 @@
         /usr/share/java
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-        /usr/share/java
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -655,7 +647,6 @@
     ${MMC_EN_JAR} ${NMCLF_JAR} ${NMCLF_EN_JAR}
     ${PKI_NSUTIL_JAR}
     ${JSS_JAR}
-    ${OSUTIL_JAR}
     ${COMMONS_CODEC_JAR})
 
 
--- a/pki/base/java-tools/src/CMakeLists.txt
+++ b/pki/base/java-tools/src/CMakeLists.txt
@@ -8,14 +8,6 @@
         /usr/share/java
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-        /usr/share/java
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -67,13 +59,13 @@
 set(CMAKE_JAVA_INCLUDE_PATH
     ${PKI_NSUTIL_JAR} ${PKI_CMSUTIL_JAR}
     ${XALAN_JAR} ${XERCES_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR})
 
 set(CMAKE_JAVA_TARGET_VERSION ${APPLICATION_VERSION})
 
 # build pki-tools
 add_jar(pki-tools ${pki-tools_java_SRCS})
-add_dependencies(pki-tools osutil pki-nsutil pki-cmsutil)
+add_dependencies(pki-tools pki-nsutil pki-cmsutil)
 install(
     FILES
         com/netscape/cmstools/DRMTool.cfg
--- a/pki/base/java-tools/templates/pki_java_command_wrapper.in
+++ b/pki/base/java-tools/templates/pki_java_command_wrapper.in
@@ -74,15 +74,12 @@
     if [ "${ARCHITECTURE}" = "i386" ] ; then
         LD_LIBRARY_PATH=/usr/lib:/lib
         LD_LIBRARY_PATH=/usr/lib/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib/osutil:${LD_LIBRARY_PATH}
         export LD_LIBRARY_PATH
     elif [ "${ARCHITECTURE}" = "x86_64" ] ; then
         LD_LIBRARY_PATH=/usr/lib:/lib
         LD_LIBRARY_PATH=/usr/lib/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib/osutil:${LD_LIBRARY_PATH}
         LD_LIBRARY_PATH=/usr/lib64:/lib64:${LD_LIBRARY_PATH}
         LD_LIBRARY_PATH=/usr/lib64/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib64/osutil:${LD_LIBRARY_PATH}
         export LD_LIBRARY_PATH
     else
         invalid_architecture "${ARCHITECTURE}"
@@ -135,13 +132,6 @@
     # Fedora 16+
     CP=/usr/lib64/java/jss4.jar
 fi
-CP=/usr/lib/java/osutil.jar:${CP}
-CP=/usr/lib/java/dirsec/osutil.jar:${CP}
-if [ "${OS}" = "Linux" ] &&
-   [ "${ARCHITECTURE}" = "x86_64" ] ; then
-    # Fedora 16+
-    CP=/usr/lib64/java/osutil.jar:${CP}
-fi
 CP=/usr/share/java/commons-codec.jar:${CP}
 CP=/usr/share/java/ldapjdk.jar:${CP}
 CP=/usr/share/java/${PRODUCT}/pki-nsutil.jar:${CP}
--- a/pki/base/java-tools/templates/pretty_print_cert_command_wrapper.in
+++ b/pki/base/java-tools/templates/pretty_print_cert_command_wrapper.in
@@ -74,15 +74,12 @@
     if [ "${ARCHITECTURE}" = "i386" ] ; then
         LD_LIBRARY_PATH=/usr/lib:/lib
         LD_LIBRARY_PATH=/usr/lib/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib/osutil:${LD_LIBRARY_PATH}
         export LD_LIBRARY_PATH
     elif [ "${ARCHITECTURE}" = "x86_64" ] ; then
         LD_LIBRARY_PATH=/usr/lib:/lib
         LD_LIBRARY_PATH=/usr/lib/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib/osutil:${LD_LIBRARY_PATH}
         LD_LIBRARY_PATH=/usr/lib64:/lib64:${LD_LIBRARY_PATH}
         LD_LIBRARY_PATH=/usr/lib64/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib64/osutil:${LD_LIBRARY_PATH}
         export LD_LIBRARY_PATH
     else
         invalid_architecture "${ARCHITECTURE}"
@@ -135,13 +132,6 @@
     # Fedora 16+
     CP=/usr/lib64/java/jss4.jar
 fi
-CP=/usr/lib/java/osutil.jar:${CP}
-CP=/usr/lib/java/dirsec/osutil.jar:${CP}
-if [ "${OS}" = "Linux" ] &&
-   [ "${ARCHITECTURE}" = "x86_64" ] ; then
-    # Fedora 16+
-    CP=/usr/lib64/java/osutil.jar:${CP}
-fi
 CP=/usr/share/java/commons-codec.jar:${CP}
 CP=/usr/share/java/ldapjdk.jar:${CP}
 CP=/usr/share/java/${PRODUCT}/pki-nsutil.jar:${CP}
--- a/pki/base/java-tools/templates/pretty_print_crl_command_wrapper.in
+++ b/pki/base/java-tools/templates/pretty_print_crl_command_wrapper.in
@@ -74,15 +74,12 @@
     if [ "${ARCHITECTURE}" = "i386" ] ; then
         LD_LIBRARY_PATH=/usr/lib:/lib
         LD_LIBRARY_PATH=/usr/lib/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib/osutil:${LD_LIBRARY_PATH}
         export LD_LIBRARY_PATH
     elif [ "${ARCHITECTURE}" = "x86_64" ] ; then
         LD_LIBRARY_PATH=/usr/lib:/lib
         LD_LIBRARY_PATH=/usr/lib/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib/osutil:${LD_LIBRARY_PATH}
         LD_LIBRARY_PATH=/usr/lib64:/lib64:${LD_LIBRARY_PATH}
         LD_LIBRARY_PATH=/usr/lib64/jss:${LD_LIBRARY_PATH}
-        LD_LIBRARY_PATH=/usr/lib64/osutil:${LD_LIBRARY_PATH}
         export LD_LIBRARY_PATH
     else
         invalid_architecture "${ARCHITECTURE}"
@@ -135,13 +132,6 @@
     # Fedora 16+
     CP=/usr/lib64/java/jss4.jar
 fi
-CP=/usr/lib/java/osutil.jar:${CP}
-CP=/usr/lib/java/dirsec/osutil.jar:${CP}
-if [ "${OS}" = "Linux" ] &&
-   [ "${ARCHITECTURE}" = "x86_64" ] ; then
-    # Fedora 16+
-    CP=/usr/lib64/java/osutil.jar:${CP}
-fi
 CP=/usr/share/java/commons-codec.jar:${CP}
 CP=/usr/share/java/ldapjdk.jar:${CP}
 CP=/usr/share/java/${PRODUCT}/pki-nsutil.jar:${CP}
--- a/pki/base/kra/src/CMakeLists.txt
+++ b/pki/base/kra/src/CMakeLists.txt
@@ -55,13 +55,6 @@
         ${JAVA_LIB_INSTALL_DIR}
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -98,7 +91,7 @@
     ${PKI_CERTSRV_JAR} ${PKI_CMS_JAR} ${PKI_CMSCORE_JAR}
     ${PKI_CMSUTIL_JAR} ${PKI_NSUTIL_JAR}
     ${LDAPJDK_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
 
 
 # set version
@@ -107,7 +100,7 @@
 
 # build pki-kra.jar
 add_jar(pki-kra ${pki-kra_java_SRCS})
-add_dependencies(pki-kra osutil symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
+add_dependencies(pki-kra symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
 install_jar(pki-kra ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_KRA_JAR ${pki-kra_JAR_FILE} CACHE INTERNAL "pki-kra jar file")
 
--- a/pki/base/ocsp/src/CMakeLists.txt
+++ b/pki/base/ocsp/src/CMakeLists.txt
@@ -55,13 +55,6 @@
         ${JAVA_LIB_INSTALL_DIR}
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -91,7 +84,7 @@
     ${PKI_CERTSRV_JAR} ${PKI_CMS_JAR} ${PKI_CMSCORE_JAR}
     ${PKI_CMSUTIL_JAR} ${PKI_NSUTIL_JAR}
     ${LDAPJDK_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
 
 
 # set version
@@ -100,7 +93,7 @@
 
 # build pki-ocsp.jar
 add_jar(pki-ocsp ${pki-ocsp_java_SRCS})
-add_dependencies(pki-ocsp osutil symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
+add_dependencies(pki-ocsp symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
 install_jar(pki-ocsp ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_OCSP_JAR ${pki-ocsp_JAR_FILE} CACHE INTERNAL "pki-ocsp jar file")
 
--- a/pki/base/scripts/pkicheck
+++ b/pki/base/scripts/pkicheck
@@ -9,5 +9,5 @@
 fi
 
 # For now, check for symkey (legacy package)
-rpm -qa | egrep -i pki-\|osutil\|symkey | sort | cat -n
+rpm -qa | egrep -i pki-\|symkey | sort | cat -n
 
--- a/pki/base/setup/pkicreate
+++ b/pki/base/setup/pkicreate
@@ -180,7 +180,6 @@
 my $jss_jar_base_name             = "jss4.jar";            # CA, KRA, OCSP, TKS
 my $ldapjdk_jar_base_name         = "ldapjdk.jar";         # CA, KRA, OCSP, TKS
 my $pki_nsutil_jar_base_name      = "pki-nsutil.jar";      # CA, KRA, OCSP, TKS
-my $osutil_jar_base_name          = "osutil.jar";          # CA, KRA, OCSP, TKS
 my $commons_codec_jar_base_name   = "commons-codec.jar";   # CA, KRA, OCSP, TKS
 my $symkey_jar_base_name          = "symkey.jar";          # CA, KRA, OCSP, TKS
 my $tomcatjss_jar_base_name       = "tomcatjss.jar";       # CA, KRA, OCSP, TKS
@@ -478,8 +477,6 @@
 my $ldapjdk_jar_symlink_path                     = undef;  # CA, KRA, OCSP, TKS
 my $pki_nsutil_jar_file_path                     = undef;  # CA, KRA, OCSP, TKS
 my $pki_nsutil_jar_symlink_path                  = undef;  # CA, KRA, OCSP, TKS
-my $osutil_jar_file_path                         = undef;  # CA, KRA, OCSP, TKS
-my $osutil_jar_symlink_path                      = undef;  # CA, KRA, OCSP, TKS
 my $commons_codec_jar_file_path                  = undef;  # CA, KRA, OCSP, TKS
 my $commons_codec_jar_symlink_path               = undef;  # CA, KRA, OCSP, TKS
 my $symkey_jar_file_path                         = undef;  # CA, KRA, OCSP, TKS
@@ -1801,11 +1798,6 @@
             return 0;
         }
 
-        if (!defined($osutil_jar_file_path = find_jar($osutil_jar_base_name))) {
-            emit("could not find jar: $osutil_jar_base_name", "error");
-            return 0;
-        }
-
         if (!defined($commons_codec_jar_file_path = find_jar($commons_codec_jar_base_name))) {
             emit("could not find jar: $commons_codec_jar_base_name", "error");
             return 0;
@@ -1882,8 +1874,6 @@
                                            . "/" . $ldapjdk_jar_base_name;
         $pki_nsutil_jar_symlink_path       = $webinf_lib_instance_path
                                            . "/" . $pki_nsutil_jar_base_name;
-        $osutil_jar_symlink_path           = $webinf_lib_instance_path
-                                           . "/" . $osutil_jar_base_name;
         $commons_codec_jar_symlink_path    = $webinf_lib_instance_path
                                            . "/" . $commons_codec_jar_base_name;
         $symkey_jar_symlink_path           = $webinf_lib_instance_path
@@ -2876,10 +2866,6 @@
         return 0 if !create_symlink($pki_nsutil_jar_symlink_path, $pki_nsutil_jar_file_path,
                                     $pki_user, $pki_group);
 
-        # create instance symlink to "osutil.jar"
-        return 0 if !create_symlink($osutil_jar_symlink_path, $osutil_jar_file_path,
-                                    $pki_user, $pki_group);
-
         # create instance symlink to "commons_codec.jar"
         return 0 if !create_symlink($commons_codec_jar_symlink_path, $commons_codec_jar_file_path,
                                     $pki_user, $pki_group);
--- a/pki/base/silent/scripts/pkisilent
+++ b/pki/base/silent/scripts/pkisilent
@@ -51,16 +51,13 @@
 if( $ARCHITECTURE eq "i386" ) {
     $libpath="/usr/lib";
 
-    $ENV{LD_LIBRARY_PATH} = "/usr/lib/osutil:"
-                          . "/usr/lib/jss:"
+    $ENV{LD_LIBRARY_PATH} = "/usr/lib/jss:"
                           . "/usr/lib:/lib";
 } elsif($ARCHITECTURE eq "x86_64") {
     $libpath="/usr/lib64";
 
-    $ENV{LD_LIBRARY_PATH} = "/usr/lib64/osutil:"
-                          . "/usr/lib64/jss:"
+    $ENV{LD_LIBRARY_PATH} = "/usr/lib64/jss:"
                           . "/usr/lib64:/lib64:"
-                          . "/usr/lib/osutil:"
                           . "/usr/lib/jss:"
                           . "/usr/lib:/lib";
 } else {
@@ -90,13 +87,10 @@
 if( $ARCHITECTURE eq "x86_64" ) {
     $ENV{CLASSPATH} = $ENV{CLASSPATH}
                     . "/usr/lib64/java/jss4.jar:"
-                    . "/usr/lib/java/jss4.jar:"
-                    . "/usr/lib64/java/osutil.jar:"
-                    . "/usr/lib/java/osutil.jar";
+                    . "/usr/lib/java/jss4.jar:";
 } else {
     $ENV{CLASSPATH} = $ENV{CLASSPATH}
-                    . "/usr/lib/java/jss4.jar:"
-                    . "/usr/lib/java/osutil.jar";
+                    . "/usr/lib/java/jss4.jar:";
 }
 
 
--- a/pki/base/silent/src/CMakeLists.txt
+++ b/pki/base/silent/src/CMakeLists.txt
@@ -76,11 +76,11 @@
     ${PKI_CERTSRV_JAR} ${PKI_CMS_JAR}
     ${PKI_CMSUTIL_JAR} ${PKI_NSUTIL_JAR}
     ${LDAPJDK_JAR} ${XERCES_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
 
 set(CMAKE_JAVA_TARGET_VERSION ${APPLICATION_VERSION})
 
 add_jar(pki-silent ${pki-silent_java_SRCS})
-add_dependencies(pki-silent osutil symkey pki-cmsutil pki-nsutil pki-certsrv pki-cms)
+add_dependencies(pki-silent symkey pki-cmsutil pki-nsutil pki-certsrv pki-cms)
 install_jar(pki-silent ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_SILENT_JAR ${pki-silent_JAR_FILE} CACHE INTERNAL "pki-silent jar file")
--- a/pki/base/tks/src/CMakeLists.txt
+++ b/pki/base/tks/src/CMakeLists.txt
@@ -55,13 +55,6 @@
         ${JAVA_LIB_INSTALL_DIR}
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -88,7 +81,7 @@
     ${PKI_CERTSRV_JAR} ${PKI_CMS_JAR} ${PKI_CMSCORE_JAR}
     ${PKI_CMSUTIL_JAR} ${PKI_NSUTIL_JAR}
     ${LDAPJDK_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR} ${SYMKEY_JAR})
 
 
 # set version
@@ -97,7 +90,7 @@
 
 # build pki-tks.jar
 add_jar(pki-tks ${pki-tks_java_SRCS})
-add_dependencies(pki-tks osutil symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
+add_dependencies(pki-tks symkey pki-nsutil pki-cmsutil pki-certsrv pki-cms pki-cmscore)
 install_jar(pki-tks ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_TKS_JAR ${pki-tks_JAR_FILE} CACHE INTERNAL "pki-tks jar file")
 
--- a/pki/base/util/src/CMakeLists.txt
+++ b/pki/base/util/src/CMakeLists.txt
@@ -16,14 +16,6 @@
         /usr/share/java
 )
 
-find_file(OSUTIL_JAR
-    NAMES
-        osutil.jar
-    PATHS
-        ${JAVA_LIB_INSTALL_DIR}
-        /usr/share/java
-)
-
 find_file(COMMONS_CODEC_JAR
     NAMES
         commons-codec.jar
@@ -338,21 +330,20 @@
 
 set(CMAKE_JAVA_INCLUDE_PATH
     ${LDAPJDK_JAR} ${XALAN_JAR} ${XERCES_JAR}
-    ${JSS_JAR} ${OSUTIL_JAR} ${COMMONS_CODEC_JAR})
+    ${JSS_JAR} ${COMMONS_CODEC_JAR})
 
 set(CMAKE_JAVA_TARGET_VERSION ${APPLICATION_VERSION})
 
 # build pki-nsutil
 set(CMAKE_JAR_CLASSES_PREFIX netscape)
 add_jar(pki-nsutil ${pki-nsutil_java_SRCS})
-add_dependencies(pki-nsutil osutil)
 install_jar(pki-nsutil ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_NSUTIL_JAR ${pki-nsutil_JAR_FILE} CACHE INTERNAL "pki-nsutil jar file")
 
 # build pki-cmsutil
 set(CMAKE_JAR_CLASSES_PREFIX com/netscape/cmsutil)
 add_jar(pki-cmsutil ${pki-cmsutil_java_SRCS})
-add_dependencies(pki-cmsutil osutil pki-nsutil)
+add_dependencies(pki-cmsutil pki-nsutil)
 install_jar(pki-cmsutil ${JAVA_JAR_INSTALL_DIR}/pki)
 set(PKI_CMSUTIL_JAR ${pki-cmsutil_JAR_FILE} CACHE INTERNAL "pki-cmsutil jar file")
 
--- a/pki/scripts/prepare_dogtag_pki
+++ b/pki/scripts/prepare_dogtag_pki
@@ -246,9 +246,6 @@
 OPENLDAP_CLIENTS="openldap-clients"
 OPENLDAP_DEVEL="openldap-devel"
 
-# Operating System Utilities
-OSUTIL="osutil"
-
 # Perl Modules
 PERL_CRYPT_SSLEAY="perl-Crypt-SSLeay"
 PERL_DBD_SQLITE="perl-DBD-SQLite"
@@ -294,7 +291,7 @@
 XML_COMMONS_RESOLVER="xml-commons-resolver"
 
 # Create a catch-all variable for PKI Support Packages
-PKI_SUPPORT_PACKAGES="${APR} ${APR_DEVEL} ${APR_UTIL} ${APR_UTIL_DEVEL} ${EXPAT} ${EXPAT_DEVEL} ${HTTPD} ${HTTPD_DEVEL} ${HTTPD_TOOLS} ${PCRE} ${PCRE_DEVEL} ${TOMCAT6} ${TOMCAT6_LIB} ${APACHE_COMMONS_LANG} ${APACHE_COMMONS_LANG} ${APACHE_COMMONS_LOGGING} ${APACHE_COMMONS_CODEC} ${JAKARTA_COMMONS_COLLECTIONS} ${JAKARTA_COMMONS_DBCP} ${JAKARTA_COMMONS_POOL} ${NSPR} ${NSPR_DEVEL} ${NSS} ${NSS_DEVEL} ${NSS_TOOLS} ${JSS} ${JSS_JAVADOC} ${TOMCATJSS} ${MOD_NSS} ${MOD_PERL} ${MOD_REVOCATOR} ${IDM_CONSOLE_FRAMEWORK} ${CYRUS_SASL} ${CYRUS_SASL_DEVEL} ${LDAPJDK} ${OPENLDAP} ${OPENLDAP_CLIENTS} ${OPENLDAP_DEVEL} ${OSUTIL} ${PERL_CRYPT_SSLEAY} ${PERL_DBD_SQLITE} ${PERL_DBI} ${PERL_HTML_PARSER} ${PERL_HTML_TAGSET} ${PERL_LIBWWW_PERL} ${PERL_MOZILLA_LDAP} ${PERL_PARSE_RECDESCENT} ${PERL_URI} ${PERL_XML_NAMESPACESUPPORT} ${PERL_XML_PARSER} ${PERL_XML_SAX} ${PERL_XML_SIMPLE} ${ESC} ${SVRCORE} ${SVRCORE_DEVEL} ${POLICYCOREUTILS} ${SELINUX_POLICY_DEVEL} ${SELINUX_POLICY_TARGETED} ${SQLITE} ${SQLITE_DEVEL} ${VELOCITY} ${BCEL} ${JAKARTA_ORO} ${JDOM} ${LOG4J} ${REGEXP} ${WERKEN_XPATH} ${XALAN_J2} ${XERCES_J2} ${XML_COMMONS_APIS} ${XML_COMMONS_RESOLVER}"
+PKI_SUPPORT_PACKAGES="${APR} ${APR_DEVEL} ${APR_UTIL} ${APR_UTIL_DEVEL} ${EXPAT} ${EXPAT_DEVEL} ${HTTPD} ${HTTPD_DEVEL} ${HTTPD_TOOLS} ${PCRE} ${PCRE_DEVEL} ${TOMCAT6} ${TOMCAT6_LIB} ${APACHE_COMMONS_LANG} ${APACHE_COMMONS_LANG} ${APACHE_COMMONS_LOGGING} ${APACHE_COMMONS_CODEC} ${JAKARTA_COMMONS_COLLECTIONS} ${JAKARTA_COMMONS_DBCP} ${JAKARTA_COMMONS_POOL} ${NSPR} ${NSPR_DEVEL} ${NSS} ${NSS_DEVEL} ${NSS_TOOLS} ${JSS} ${JSS_JAVADOC} ${TOMCATJSS} ${MOD_NSS} ${MOD_PERL} ${MOD_REVOCATOR} ${IDM_CONSOLE_FRAMEWORK} ${CYRUS_SASL} ${CYRUS_SASL_DEVEL} ${LDAPJDK} ${OPENLDAP} ${OPENLDAP_CLIENTS} ${OPENLDAP_DEVEL} ${PERL_CRYPT_SSLEAY} ${PERL_DBD_SQLITE} ${PERL_DBI} ${PERL_HTML_PARSER} ${PERL_HTML_TAGSET} ${PERL_LIBWWW_PERL} ${PERL_MOZILLA_LDAP} ${PERL_PARSE_RECDESCENT} ${PERL_URI} ${PERL_XML_NAMESPACESUPPORT} ${PERL_XML_PARSER} ${PERL_XML_SAX} ${PERL_XML_SIMPLE} ${ESC} ${SVRCORE} ${SVRCORE_DEVEL} ${POLICYCOREUTILS} ${SELINUX_POLICY_DEVEL} ${SELINUX_POLICY_TARGETED} ${SQLITE} ${SQLITE_DEVEL} ${VELOCITY} ${BCEL} ${JAKARTA_ORO} ${JDOM} ${LOG4J} ${REGEXP} ${WERKEN_XPATH} ${XALAN_J2} ${XERCES_J2} ${XML_COMMONS_APIS} ${XML_COMMONS_RESOLVER}"
 
 ###########################################
 # Establish PKI Installation Dependencies #
--- a/pki/scripts/remove_pki_components
+++ b/pki/scripts/remove_pki_components
@@ -108,14 +108,7 @@
     # (1) grab all PKI components
     PKI_COMPONENTS=`rpm -qa --queryformat '%{NAME}\n' | grep pki`
 
-    # (2) check for osutil
-    `rpm -q --quiet osutil`
-    OSUTIL_PRESENCE=$?
-    if [ "${OSUTIL_PRESENCE}" = "0" ] ; then
-        PKI_COMPONENTS="${PKI_COMPONENTS} osutil"
-    fi
-
-    # (3) check for symkey (legacy package)
+    # (2) check for symkey (legacy package)
     `rpm -q --quiet symkey`
     SYMKEY_PRESENCE=$?
     if [ "${SYMKEY_PRESENCE}" = "0" ] ; then
--- a/pki/specs/dogtag-pki.spec
+++ b/pki/specs/dogtag-pki.spec
@@ -73,7 +73,6 @@
 # Make certain that this 'meta' package requires the latest version(s)
 # of ALL top-level Dogtag PKI support packages
 Requires:         jss >= %{jss_version}
-Requires:         osutil >= %{osutil_version}
 Requires:         tomcatjss >= %{tomcatjss_version}
 
 # Make certain that this 'meta' package requires the latest version(s)
--- a/pki/specs/pki-core.spec
+++ b/pki/specs/pki-core.spec
@@ -190,16 +190,13 @@
 %if 0%{?fedora} >= 16
 Requires:         jpackage-utils >= 0:1.7.5-10
 Requires:         jss >= 4.2.6-19.1
-Requires:         osutil >= 2.0.2
 %else
 %if 0%{?fedora} >= 15
 Requires:         jpackage-utils
 Requires:         jss >= 4.2.6-17
-Requires:         osutil >= 2.0.1
 %else
 Requires:         jpackage-utils
 Requires:         jss >= 4.2.6-17
-Requires:         osutil
 %endif
 %endif
 
--- a/pki/tools/jar/README.jar-tools
+++ b/pki/tools/jar/README.jar-tools
@@ -225,9 +225,8 @@
 The summary is listed below:
 
 Summary:
-22 Unique Jar's
+21 Unique Jar's
      /usr/lib/jss/jss4-4.2.6.jar
-     /usr/lib/osutil/osutil-2.0.0.jar
      /usr/share/java/commons-codec.jar
      /usr/lib/symkey/symkey-9.0.0.jar
      /usr/share/java/jakarta-taglibs-core-1.1.1.jar
@@ -248,11 +247,10 @@
      /usr/share/java/xerces-j2-2.9.0.jar
      /usr/share/java/xml-commons-apis-1.4.01.jar
      /usr/share/java/xml-commons-apis-ext-1.4.01.jar
-16 Unique RPM's
+15 Unique RPM's
      jakarta-taglibs-standard
      jss
      ldapjdk
-     osutil
      apache-commons-codec
      pki-common
      pki-console
@@ -266,13 +264,13 @@
      xerces-j2
      xml-commons-apis
 
-What this is telling us is that there are 22 jars which provide all
+What this is telling us is that there are 21 jars which provide all
 the classes needed to satisfy the import statements. However there may
 be some duplicates in the list. Also because of wildcard import
 statements some classes and hence jars may have been included which
 are not actually utilized in the code.
 
-Those 22 jars are provided by the 16 RPM's listed. Once again there
+Those 21 jars are provided by the 15 RPM's listed. Once again there
 may be some class duplication and/or unnecessary RPM's due to
 wildcarding. But this gives a very small manageable list to manually
 pick through and make our choices. For example, one thing we can
